.azure/OneBranch.Package.yml

```diff
@@ -23,111 +32,101 @@ parameters:
-  - template: ./obtemplates/push-vpack.yml
-    parameters:
-      platform: amd64fre
-
-- stage: push_amd64chk
-  displayName: Push amd64chk VPack
-  dependsOn: [prepare]
-  jobs:
-  - template: ./obtemplates/push-vpack.yml
-    parameters:
-      platform: amd64chk
-
-- stage: push_arm64fre
-  displayName: Push arm64fre VPack
-  dependsOn: [prepare]
-  jobs:
-  - template: ./obtemplates/push-vpack.yml
-    parameters:
-      platform: arm64fre
-
-- stage: push_arm64chk
-  displayName: Push arm64chk VPack
-  dependsOn: [prepare]
-  jobs:
-  - template: ./obtemplates/push-vpack.yml
-    parameters:
-      platform: arm64chk
-
-- stage: push_armfre
-  displayName: Push armfre VPack
-  dependsOn: [prepare]
-  jobs:
-  - template: ./obtemplates/push-vpack.yml
-    parameters:
-      platform: armfre
-
-- stage: push_armchk
-  displayName: Push armchk VPack
-  dependsOn: [prepare]
-  jobs:
-  - template: ./obtemplates/push-vpack.yml
-    parameters:
-      platform: armchk
-
-- stage: push_chpefre
-  displayName: Push chpefre VPack
-  dependsOn: [prepare]
-  jobs:
-  - template: ./obtemplates/push-vpack.yml
-    parameters:
-      platform: chpefre
-
-- stage: push_chpechk
-  displayName: Push chpechk VPack
-  dependsOn: [prepare]
-  jobs:
-  - template: ./obtemplates/push-vpack.yml
-    parameters:
-      platform: chpechk
-
-- stage: push_x86fre
-  displayName: Push x86fre VPack
-  dependsOn: [prepare]
-  jobs:
-  - template: ./obtemplates/push-vpack.yml
-    parameters:
-      platform: x86fre
-
-- stage: push_x86chk
-  displayName: Push x86chk VPack
-  dependsOn: [prepare]
-  jobs:
-  - template: ./obtemplates/push-vpack.yml
-    parameters:
-      platform: x86chk
-
-- stage: push_git
-  displayName: Push to Git
-  dependsOn:
-  - push_amd64fre
-  - push_amd64chk
-  - push_arm64fre
-  - push_arm64chk
-  - push_armfre
-  - push_armchk
-  - push_chpefre
-  - push_chpechk
-  - push_x86fre
-  - push_x86chk
-  jobs:
-  - template: ./obtemplates/git-submit.yml
+  ROOT: $(Build.SourcesDirectory)
+  REPOROOT: $(Build.SourcesDirectory)
+  OUTPUTROOT: $(REPOROOT)\out
+  NUGET_XMLDOC_MODE: none
+  WindowsContainerImage: 'onebranch.azurecr.io/windows/ltsc2022/vse2022:latest'
+
+extends:
+  template: v2/Microsoft.Official.yml@templates # https://aka.ms/obpipelines/templates
+  parameters:
+    platform:
+      name: 'windows_undocked' # windows undocked
+    cloudvault: # https://aka.ms/obpipelines/cloudvault
+      enabled: false
+    globalSdl: # https://aka.ms/obpipelines/sdl
+      tsa:
+        enabled: false # onebranch publish all sdl results to TSA. If TSA is disabled all SDL tools will forced into 'break' build mode.
+      credscan:
+        suppressionsFile: $(Build.SourcesDirectory)\.azure\CredScanSuppressions.json
+      binskim:
+        break: true # always break the build on binskim issues in addition to TSA upload
+      policheck:
+        break: true # always break the build on policheck issues. You can disable it by setting to 'false'
```

Comment:
I believe this are actually the defaults, so they aren't needed either.

Reply:
Are you sure credscan and supppressionsFile are also the defaults?

Reply:
I saw you highlighted only 4 lines. So, they are removed now.